frontend/src/entities/configuration/dynamic-configuration.tsx

```diff
@@ -145,7 +146,14 @@ export function DynamicConfiguration () {
                     />
                 </ExpandableSection>
                 <ExpandableSection headerText='Project Creation' variant='default' defaultExpanded>
-                    {<pre>TODO</pre>}
+                    <Toggle
+                        onChange={({ detail }) => {
+                            setFields({ 'configuration.ProjectCreation.isAdminOnly': detail.checked });
+                        }}
+                        checked={state.form.configuration.ProjectCreation?.isAdminOnly || false}
+                    >
+                        Admin Only - when enabled, only admins will be able to create new projects.
```

Comment:
Text seems a bit duplicative. Not sold on this:
```suggestion
                        Admin Only - restrict creation of projects to users with the Admin permission.
```

Reply:
Your suggestion seems fine, maybe "restrict creation of projects to users with Admin permissions" but same difference